Fix: Validate PrivacyInfo.xcprivacy and warn users of required missing privacy parameters. (#152)
getsentry/sentry-xamarin: CHANGELOG.md

# Changelog

## Unreleased

### Fixes

- Validate PrivacyInfo.xcprivacy and warn users of required missing privacy parameters ([#152](https://github.com/getsentry/sentry-xamarin/pull/152))

>   Due to a requirement by apple, all Projects will require a ([Privacy Manifest file](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files)). Sentry will automatically patch your project with the required fields by Sentry if no privacy manifest file is provided. Otherwise, it will validate your privacy manifest file for the required fields by Sentry and warn you during the build time if the required keys are present or not when building an iOS project.

Reviewer: ```suggestion
  Due to new Apple requirements, all Projects will require a ([Privacy Manifest file](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files)). Sentry will automatically patch your project with the fields required by Sentry, if no privacy manifest file is provided. Otherwise, it will validate your privacy manifest file and warn during the build if the required keys are not when building an iOS project.
```

>   Below you can find a snippet of the minimum required privacy manifest required by Sentry Xamarin:

Reviewer: ```suggestion
  Below you can find a snippet of the minimum privacy manifest required by Sentry Xamarin:
```
